mono/mini/mini-posix.c

```diff
@@ -776,7 +770,7 @@ mono_runtime_shutdown_stat_profiler (void)
 {
 	InterlockedWrite (&sampling_thread_running, 0);
 
-#ifdef HAVE_CLOCK_NANOSLEEP
+#ifndef PLATFORM_MACOSX
```

Comment:
For whatever strange reason, this macro is defined for all Apple platforms. It should be renamed at some point.